cookbooks/kinematically_driven_subduction_2d/composition_trms_statistics.cc

```diff
@@ -0,0 +1,227 @@
+/*
+  Copyright (C) 2011, 2012 by the authors of the ASPECT code.
```

Comment:
in all the files containing this copyright line, there is no unity with regards to the years.